galsim/phase_psf.py

```diff
@@ -1247,6 +1252,9 @@ class OpticalPSF(GSObject):
                             individual aberration.  Note that aberrations[1] is piston (and not
                             aberrations[0], which is unused.)  This list can be arbitrarily long to
                             handle Zernike polynomial aberrations of arbitrary order.
+    @param annular_zernike  Boolean indicating that aberrations specify the amplitudes of annular
```

Comment:
My intent was for both named modes and `aberrations` would both be available for both circular and annular Zernikes.  I'm not sure if there's a standard in the literature for this or not, but it certainly doesn't break any of the math.